docs/pages/management/operations/db-ca-rotation.mdx

```diff
@@ -0,0 +1,210 @@
+
+### `db` CA Rotation phases
+
+- `standby`: All operations have completed or haven't started yet.
+- `init`: All Teleport cluster services are notified of the rotation.
+  A new `db` certificate authority is created in your cluster.
+  Remote trusted clusters must fetch the new CA.
+  `tctl auth sign` will issue database server certs with the new CA.
+- `update_clients`: Ephemeral client certificates issued to the Teleport Proxy
+  Service will be issued by the new CA.
+- `update_servers`: No effect during `db` CA rotation.
+- `rollback`: The rotation was aborted and is rolling back to the old
+  certificate authority.
+
+### `db_client` CA Rotation phases
+
+- `standby`: All operations have completed or haven't started yet.
+- `init`: All Teleport cluster services are notified of the rotation.
+  A new `db_client` certificate authority is created in your cluster.
+  `tctl auth sign` will output both the old and new certificate authorities
+  in its trusted CA output.
+- `update_clients`: Ephemeral client certificates issued to the Teleport 
+  Database Service will be issued by the new CA.
+- `update_servers`: No effect during `db_client` CA rotation.
+- `rollback`: The rotation was aborted and is rolling back to the old
+  certificate authority.
+
+### Rotation types
+
+There are two types of certificate authority rotation processes:
+
+- **Manual:** it is the cluster administrator's responsibility to transition
+  between each phase of the rotation while monitoring database connectivity.
+  Manual rotations are performed by providing the `--manual` flag and the
+  desired phase using the `--phase` flag to the `tctl auth rotate` command.
+- **Semi-automatic:** Teleport automatically transitions between phases of the
+  rotation after some amount of time (known as a *grace period*) elapses.
+
+For both types of rotation, the cluster goes through the phases in the
+following order:
+
+- `standby` -> `init` -> `update_clients` -> `update_servers` -> `standby`
+  
+It does not make sense to use semi-automatic rotation when rotating the `db`
+or `db_client` CA, because self-hosted databases are not automatically
+reconfigured by the semi-automatic process.
+Therefore this guide will only explain manual rotation.
+
+In manual mode we transition between phases while monitoring database
+connectivity, and reconfigure databases with new CA certificates as needed.
+
+You will need to reconfigure any given self-hosted database only once during the
+rotation: it does not need to be done at each step.
+You can reconfigure your databases during any phase, although it's best to do
+so during the `init` phase to avoid losing database access later in the rotation.
+
+If you lose connectivity to your databases during any phase prior to
+`standby`, you can [roll back](#rollback) to the old certificate authority to
+restore connectivity.
+
+## Step 1/4. Start the rotation
+
+Initiate manual rotation of the `db` or `db_client` certificate authority:
```

Comment:
Hmm. I figured it would be simpler to rotate one CA at a time.
However, I guess there's nothing wrong with rotating both `db` and `db_client` CA certs at the same time (I wouldn't do `host` CA at the same time as either because host CA rotations have a much bigger impact on intra-cluster connectivity).

I'll update both the migrations and the db ca rotation guide to clarify that it's ok to rotate `db` and `db_client` at the same time.

Reply:
058e472620fe90b5993b644cab5aa51aadf5415a